### PR TITLE
feat: improved integration tests

### PR DESCRIPTION
**Pull request by ratik**
This one contains of description of new integration tests setup and components

### CODE REVIEW

**Review by albertandrejev — DISMISSED** [review]

File: docs/tutorials/new_integration_tests.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+7. `yarn build-images` - Build docker images for the networks used
+7. `yarn build-ts-client` - Build TS client for your contracts
+8. Update the tests in `src/testcases` folder
+9. `yarn test`
+
+## What's inside the tests
+
+`testSuite.ts` contains of configuration of the networks used with defined network params and docker image names.
+
+`testcases` folder contains of the tests for the contracts. Each test is a separate file with a set of tests for a contract. Please check the existing tests to understand how to write your own. In the `beforeAll` method you can see the configuration of the networks and the deployment of the contracts.
```
Comment (sotnikov-s) [inline docs/tutorials/new_integration_tests.md:30]:
```suggestion
`testcases` folder contains the tests for the contracts. Each test is a separate file with a set of tests for a contract. Please check the existing tests to understand how to write your own. In the `beforeAll` method you can see the configuration of the networks and the deployment of the contracts.
```

File: sidebars.js [hunk 2]
```diff
@@ -335,6 +335,7 @@ const sidebars = {
                 'tutorials/cosmwasm_ica',
                 'tutorials/cosmwasm_icq',
                 'tutorials/integration_tests',
+                'tutorials/new_integration_tests',
```
Comment (sotnikov-s) [inline sidebars.js:338]: @pr0n00gler @swelf19 I think we can simply replace the original `tutorials/integration_tests` page with the new one. The old one suggests to use the quite old-fashioned, overloaded with our specific tests `neutron-integration-tests` repo which I think is more for our own usage. Strangers don't deserve such a suffering. And the new one is quite clean and neat with cool vitest under the hood. What do you think?

Comment (sotnikov-s) [reply sidebars.js:338]: Another approach here is to keep both pages for a while until we're confident about the new suggested approach (e.g. I didn't run the new workflow, not even sure it works). The `demo-integration-tests` is also poorly documented — don't think it's ready to be used as the only proper documentation for integration tests.

Comment (sotnikov-s) [inline sidebars.js:338]: 1. the header is used as the page name on the sidebar and it's too long for it
2. new_integration_tests is a bit odd name. let's give it another name?
image

Comment (sotnikov-s) [reply sidebars.js:338]: another idea is we could add a folder and move the files inside it:

Tutorials
-- Overview
-- Introduction to CosmWasm
-- ...
-- Integration tests
---- Smart contracts (the new one)
---- Legacy (the old one)

**Review by sotnikov-s — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 2 above)
Comment (ratik) [reply sidebars.js]: tried to address it. please check if it fits better now

**Review by zavgorodnii — DISMISSED** [review]

**Review by sotnikov-s — APPROVED** [review]

**Review by pr0n00gler — APPROVED** [review]

**Pull request merged**